package.json

```diff
@@ -84,7 +86,9 @@
 		"collectCoverageFrom": [
 			"src/**/*.test.{js,jsx,ts,tsx}",
```

Comment:
Is there a reason the original change we made in this PR for this line was reverted?  Test files should not be included in the code coverage. Running with this line as is artificially boosts our code coverage %

Reply:
Along the same lines see:

https://github.com/NCIOCPL/clinical-trials-search-app/issues/265#issuecomment-898046117

We initially had a change in this PR to address this as well

Reply:
@belaolsonNIH This can be reverted after going with the update @adriancofie noted above from CTS.